src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -161,12 +174,15 @@ private function processNodeWithType(NodeInterface $node, NodeType $nodeTypeDefi
             }
 
             if (!$node->hasProperty($propertyDef->getName())) {
-                if ($propertyDef->isMandatory() && !$propertyDef->isAutoCreated()) {
+                if ($propertyDef->isMandatory()
+                    && !$propertyDef->isAutoCreated()
+                    && $nodeTypeDefinition->getName() !== VersionHandler::MIX_VERSIONABLE
```

Comment:
That code is actually already in #285... I don't think that it will be too easy to review this one before the other one is merged :-) But I can change this if the others agree.